.github/workflows/docker.yml

```diff
@@ -31,9 +31,14 @@ jobs:
           compose-file: ./docker-compose.yaml
           services: base
           up-flags: --build
+          cwd: ${{ github.workspace }}
       # - name: Verifying the container was built
       #   run: docker exec roscon24-workshop-base-1 colcon test
```

Comment:
These 2 lines can probably be deleted since it's definitely using the container in the "Test pyrobosim" step

Reply:
The error is a `CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage`

`flexbe_testing` is using an older CMakeLists.txt setup , but it is pure Python, so it should not need a C compiler.  I have never seen that issue before, so there must be some deep configuration issue.

Can we just set that environment variable and see what happens? `set(CMAKE_C_COMPILER /path/to/gcc)`

Reply:
Thanks! I think I found out this error was a red herring for CI having two concurrent `colcon build`s going in its previous state. Seems fine now.